modules/vpc/variables.tf

```diff
@@ -14,6 +14,11 @@ variable "kubernetes_cluster_name" {
   description = "Kubernetes cluster name used to associate with subnets for auto LB placement"
 }
 
+variable "enable_nat_gateway" {
+  description = "Enable nat-gateway"
```

Comment:
```suggestion
  description = "Create NAT gateway(s) to allow private subnets to route traffic out to the public internet. If this is set to false, it will create a NAT instance instead. This can be useful in non-production environments to reduce cost, though in some cases it may lead to network instability or lower throughput."
```